### ISSUE TITLE
Account suspended / rate limited

### ISSUE THREAD

**Issue by LilaRest** [junhoyeo/threads-api#143]
Hi! 
After a few requests using this library my account have been suspended.
I only followed 10-20 accounts and liked 40-50 threads on a 30 minutes time span.

Any one experiencing the same thing or have been rate limited?

I create this issue so we can try to figure out why this happen in order to avoid it. :upside_down_face:

**Comment by francistogram** [comment]
Hadn’t realized they were cracking down already

Is this a full suspension or temporary? Hopefully you get your account back

**Comment by cedriking** [comment]
Was this a brand new account? Are you using a VPN? Seems your interactions were small to be banned so fast.

**Comment by LilaRest** [comment]
@cedriking Yeah a brand new account, and no I'm not using a VPN to request the API

**Comment by cedriking** [comment]
Instagram is pretty strict on brand new accounts, so maybe this is similar on Threads and the reason of your account being banned. I tried manually with a brand new account and got rate limited, but only for 2 hours, so I know I cannot use `threads-api` on that one or it'll be banned.

**Comment by LilaRest** [comment]
@cedriking Indeed. For anyone interested here, I've tried again with a brand new account but have manually posted and interacted 15-20 times with other threaders. This time I just got rate limited on my IP after 24 hours, using another IP works well.

**Comment by aleclarson** [comment]
In-app requests include `navChain` like the following:
```
"BarcelonaNavigationLogger$logNavigationCompleted$1:ig_text_feed_timeline:1:cold_start:1689533223.537::,BarcelonaNavigationLogger$logNavigationCompleted$1:ig_text_feed_timeline:2:button:1689533223.998::,BarcelonaNavigationLogger$logNavigationCompleted$1:ig_text_feed_composer:3:button:1689533229.278::,TRUNCATEDx9,BarcelonaNavigationLogger$logNavigationCompleted$1:ig_text_feed_timeline:13:button:1689533666.796::,BarcelonaNavigationLogger$logNavigationCompleted$1:ig_text_feed_composer:14:button:1689533700.682::,BarcelonaNavigationLogger$logNavigationCompleted$1:ig_text_feed_timeline:15:button:1689534265.189::,BarcelonaNavigationLogger$logNavigationCompleted$1:ig_text_feed_timeline:16:button:1689534265.220::,BarcelonaNavigationLogger$logNavigationCompleted$1:ig_text_feed_composer:17:button:1689534267.102::,BarcelonaNavigationLogger$logNavigationCompleted$1:ig_text_feed_timeline:18:button:1689534293.345::,BarcelonaNavigationLogger$logNavigationCompleted$1:ig_text_feed_timeline:19:button:1689534293.372::"
```

It's possible that is used in detection of foreign clients.

**Comment by cedriking** [comment]
Be careful out there spamming. Remember that spamming is not alright, and it's not the purpose of this API. 

Also:
![Screenshot_20230717_130141_Threads.jpg](https://github.com/junhoyeo/threads-api/assets/33100863/2a8095bc-de95-44a8-9be7-d45f7fcb3def)
